bench/group.bench.ts

```diff
@@ -21,8 +21,8 @@ export async function benchGroup(bs: Benchmark.Suite) {
     const msg = te.encode('msg')
     const dst = te.encode('dst')
 
-    for (const id of Oprf.Group.supportedGroups) {
-        const gg = Oprf.Group.fromID(id)
+    for (const id of Crypto.Group.supportedGroups) {
```

Comment:
please check the `benchOPRF` function, I was trying to run something like this:

```ts
const bs1 = new Benchmark.Suite;
Oprf.Crypto = CryptoSjcl;
benchOPRF(bs1);

const bs2 = new Benchmark.Suite;
Oprf.Crypto = CryptoNoble;
benchOPRF(bs2)

```
all these in the same function, but have no success.
I tried to patch by passing the `provider` to the function, and worked. But I didn't like it so much. 
The idea is to setup/change the provider at a higher level.

Reply:
Note that now Oprf does not have a Crypto, or Group member.

Set Crypo.provider = CryptoNoble to override

Reply:
> The idea is to setup/change the provider at a higher level.

I am not quite following sorry

Reply:
Oh, I see, on it
Updating benches. Also some problems with needing to do support groups.
```
(base) ➜  voprf-ts git:(nd-reveal-support-for-noble-crypto-backend-2023-10-04) ✗ npm run bench

> @cloudflare/voprf-ts@0.21.2 bench
> tsc -b bench && node ./lib/esm/bench/index.js

Error: group: bad group name ristretto255.
Stack: Error: group: bad group name ristretto255.
```

Reply:
```
(base) ➜  voprf-ts git:(nd-reveal-support-for-noble-crypto-backend-2023-10-04) npm run bench                    

> @cloudflare/voprf-ts@0.21.2 bench
> tsc -b bench && node ./lib/esm/bench/index.js

noble/OPRF/P256-SHA256/blind     x 470 ops/sec ±0.31% (92 runs sampled)
noble/OPRF/P256-SHA256/blindEval x 525 ops/sec ±0.44% (88 runs sampled)
noble/OPRF/P256-SHA256/finalize  x 521 ops/sec ±0.35% (91 runs sampled)
noble/OPRF/P384-SHA384/blind     x 184 ops/sec ±0.50% (87 runs sampled)
noble/OPRF/P384-SHA384/blindEval x 210 ops/sec ±0.50% (90 runs sampled)
noble/OPRF/P384-SHA384/finalize  x 210 ops/sec ±0.36% (90 runs sampled)
noble/OPRF/P521-SHA512/blind     x 90.67 ops/sec ±1.03% (86 runs sampled)
noble/OPRF/P521-SHA512/blindEval x 106 ops/sec ±0.47% (84 runs sampled)
noble/OPRF/P521-SHA512/finalize  x 106 ops/sec ±0.23% (84 runs sampled)
noble/OPRF/ristretto255-SHA512/blind     x 771 ops/sec ±0.23% (90 runs sampled)
noble/OPRF/ristretto255-SHA512/blindEval x 899 ops/sec ±0.26% (91 runs sampled)
noble/OPRF/ristretto255-SHA512/finalize  x 826 ops/sec ±0.16% (92 runs sampled)
noble/OPRF/decaf448-SHAKE256/blind     x 278 ops/sec ±0.24% (86 runs sampled)
```

Reply:
```
sjcl/OPRF/P256-SHA256/blind     x 47.21 ops/sec ±0.26% (76 runs sampled)
sjcl/OPRF/P256-SHA256/blindEval x 133 ops/sec ±0.19% (90 runs sampled)
sjcl/OPRF/P256-SHA256/finalize  x 130 ops/sec ±0.36% (88 runs sampled)
sjcl/OPRF/P384-SHA384/blind     x 33.80 ops/sec ±0.25% (81 runs sampled)
sjcl/OPRF/P384-SHA384/blindEval x 99.97 ops/sec ±0.23% (80 runs sampled)
sjcl/OPRF/P384-SHA384/finalize  x 99.46 ops/sec ±0.31% (80 runs sampled)
sjcl/OPRF/P521-SHA512/blind     x 15.85 ops/sec ±0.18% (77 runs sampled)
sjcl/OPRF/P521-SHA512/blindEval x 50.14 ops/sec ±0.16% (80 runs sampled)
sjcl/OPRF/P521-SHA512/finalize  x 48.37 ops/sec ±3.83% (79 runs sampled)
sjcl/VOPRF/P256-SHA256/blind     x 47.86 ops/sec ±0.20% (77 runs sampled)
sjcl/VOPRF/P256-SHA256/blindEval x 17.99 ops/sec ±0.26% (85 runs sampled)
sjcl/VOPRF/P256-SHA256/finalize  x 14.06 ops/sec ±0.51% (69 runs sampled)
sjcl/VOPRF/P384-SHA384/blind     x 33.96 ops/sec ±0.26% (81 runs sampled)
sjcl/VOPRF/P384-SHA384/blindEval x 13.33 ops/sec ±0.25% (66 runs sampled)
sjcl/VOPRF/P384-SHA384/finalize  x 10.11 ops/sec ±0.67% (52 runs sampled)
sjcl/VOPRF/P521-SHA512/blind     x 15.69 ops/sec ±0.32% (76 runs sampled)
sjcl/VOPRF/P521-SHA512/blindEval x 6.44 ops/sec ±0.47% (35 runs sampled)
sjcl/VOPRF/P521-SHA512/finalize  x 4.79 ops/sec ±1.07% (28 runs sampled)
sjcl/POPRF/P256-SHA256/blind     x 47.40 ops/sec ±0.27% (76 runs sampled)
sjcl/POPRF/P256-SHA256/blindEval x 15.14 ops/sec ±0.26% (74 runs sampled)
sjcl/POPRF/P256-SHA256/finalize  x 12.28 ops/sec ±0.44% (62 runs sampled)
sjcl/POPRF/P384-SHA384/blind     x 33.45 ops/sec ±0.30% (80 runs sampled)
sjcl/POPRF/P384-SHA384/blindEval x 10.94 ops/sec ±0.44% (56 runs sampled)
```